Lib/test/test_ssl.py

```diff
@@ -33,6 +34,13 @@
 def data_file(*name):
     return os.path.join(os.path.dirname(__file__), *name)
 
+def _can_decode_unicode(unicode_str):
+    try:
+        unicode_str.decode()
+        return True
+    except exceptions.UnicodeEncodeError:
```

Comment:
There is no need to use the exceptions module: "UnicodeEncodeError" alone works too, it's a builtin.